<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnexionDialog</class>
 <widget class="QDialog" name="ConnexionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>461</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="hostLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>62</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="logLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>62</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="passLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>62</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="logEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnexionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnexionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
